lib/ASTGen/Sources/ASTGen/CompilerBuildConfiguration.swift

```diff
@@ -150,6 +157,18 @@ struct CompilerBuildConfiguration: BuildConfiguration {
   }
 }
 
+enum IfConfigError: Error, CustomStringConvertible {
+  case unexpectedRuntimeCondition
+
+  var description: String {
+    switch self {
+      case .unexpectedRuntimeCondition:
+        return "unexpected argument for the '_runtime' condition; expected '_Native' or '_ObjC'"
```

Comment:
Isn’t `_multithreaded` also allowed, based on the check above?

Reply:
Heh, yes. I was originally following the actually-incorrect diagnostic text in the compiler itself to try to minimize the diffs in this PR. We can fix this now.